src/Resources/public/reference-entity/record/table/value/value-updater.ts

```diff
@@ -34,6 +37,32 @@ export default class ValueUpdater {
         }
     }
 
+    public createDragOver(): (event: React.DragEvent<HTMLTableRowElement>) => void {
+        return (event: React.DragEvent<HTMLTableRowElement>) => {
+            event.preventDefault();
+        };
+    }
+
+    public createDragStart(index: number): (event: React.DragEvent<HTMLTableRowElement>) => void {
+        return () => {
+            this.dragPosition = index;
+        };
+    }
+
+    public createDrop(index: number): (event: React.DragEvent<HTMLTableRowElement>) => void {
+        return (event: React.DragEvent<HTMLTableRowElement>) => {
+            event.preventDefault();
+
+            const row: TableDataRow | undefined = this.internalDataRows.splice(this.dragPosition, 1).shift();
+            if (row === undefined) {
```

Comment:
then here is just `if (!row)`

Reply:
prettify creates `if (!row) return;`. :+1: